aplus/en/formats.py

```diff
@@ -1,2 +1,6 @@
 
-DATETIME_FORMAT = 'D, M j Y, P'
+DATE_FORMAT = 'j F, Y'
+TIME_FORMAT = 'H:i'
+DATETIME_FORMAT = 'l, j F Y, H:i'
+SHORT_DATE_FORMAT = 'j.n.Y'
+SHORT_DATETIME_FORMAT = 'j.n.Y H:i'
```

Comment:
We need to also define following variables for the `en` file, though I'm not sure if these matter at all.

```python
MONTH_DAY_FORMAT = 'j F' # not currently used, so this isn't so imporant
FIRST_DAY_OF_WEEK = 1 # I'm not sure if this is used, but the en default is Sunday
DECIMAL_SEPARATOR = ',' # I'm not sure about this, but the default is . for en and , for fi
THOUSAND_SEPARATOR = '\xa0'  # Non-breaking space  - I'm not sure if about this, but default is , for en and nbsp for fi
```

And following for both (fi is missing some iso stuff)

```python
DATE_INPUT_FORMATS = [
    '%d.%m.%Y', '%d.%m.%y',             # '20.3.2014'. '20.3.14'
    '%d/%m/%Y', '%d/%m/%y',             # '25/10/2006', '25/10/06'
    '%Y-%m-%d',                         # '2006-10-25'
]
DATETIME_INPUT_FORMATS = [
    '%d.%m.%Y %H.%M.%S',                # '20.3.2014 14.30.59'
    '%d.%m.%Y %H.%M.%S.%f',             # '20.3.2014 14.30.59.000200'
    '%d.%m.%Y %H.%M',                   # '20.3.2014 14.30'
    '%d.%m.%y %H.%M.%S',                # '20.3.14 14.30.59'
    '%d.%m.%y %H.%M.%S.%f',             # '20.3.14 14.30.59.000200'
    '%d.%m.%y %H.%M',                   # '20.3.14 14.30'
    '%Y-%m-%d %H:%M:%S',                # '2006-10-25 14:30:59'
    '%Y-%m-%d %H:%M:%S.%f',             # '2006-10-25 14:30:59.000200'
    '%Y-%m-%d %H:%M',                   # '2006-10-25 14:30'
    '%d/%m/%Y %H:%M:%S',                # '25/10/2006 14:30:59'
    '%d/%m/%Y %H:%M:%S.%f',             # '25/10/2006 14:30:59.000200'
    '%d/%m/%Y %H:%M',                   # '25/10/2006 14:30'
    '%d/%m/%y %H:%M:%S',                # '25/10/06 14:30:59'
    '%d/%m/%y %H:%M:%S.%f',             # '25/10/06 14:30:59.000200'
    '%d/%m/%y %H:%M',                   # '25/10/06 14:30'
]
```

See these for reference

* fi: https://github.com/django/django/blob/master/django/conf/locale/fi/formats.py (will be used for values not in our file)
* en: https://github.com/django/django/blob/master/django/conf/locale/en/formats.py (will be used for values not in our file)
* en_GB: https://github.com/django/django/blob/master/django/conf/locale/en_GB/formats.py (reference values for GB, i.e., these would be better defaults)

Reply:
Don't we want all the time formats to be `14:30` and not `14.30`?
Also what is the purpose of defining these formats with slashes, such as `25/10/06 14:30`?

Reply:
These are possible strings in "date" input fields for forms. Basically, every posibility that the browsers could send should be here to my knowledge. Hence, we should have variations of the first 6 with `:` in them. And `dd/mm/yyyy` is valid format in Finland, so I guess we need to be able to parse it.